### ISSUE TITLE
Number of classes of domain classifier

### ISSUE THREAD

**Issue by ilteralp** [fungtion/DANN#16]
Hi all, I have a question. Why does the output layer of domain classifier have 2 output features ([this line](https://github.com/fungtion/DANN/blob/master/models/model.py#L35)) ? Since the domain labels are either `0` or `1` in the paper, I think it should be a binary classifier as below,

`self.domain_classifier.add_module('d_fc2', nn.Linear(100, 1))`

**Comment by msseibel** [comment]
Agree. But weirdly I saw this also in many other implementations.

**Comment by GZK66666** [comment]
> 
The target label is a binary one-hot vector, so there are two outputs, and finally through softmax？

**Comment by fungtion** [comment]
@ilteralp Yes, you can set the number as 1, I used 2 outputs for multiple domains purpose
